modules/managing-namespace-auto-pruning-policies.adoc

```diff
@@ -232,16 +223,236 @@ $ curl -X GET -H "Authorization: Bearer <access_token>" http://<quay-server.exam
 {"policies": [{"uuid": "8c03f995-ca6f-4928-b98d-d75ed8c14859", "method": "number_of_tags", "value": 10}]}
 ----
 
-. You can delete the auto-prune policy by entering the following command. Note that deleting the policy require the UUID.
+. You can delete the auto-prune policy by entering the following command. Note that deleting the policy requires the UUID.
 +
 [source,terminal]
 ----
-$ curl -X DELETE -H "Authorization: Bearer <access_token>" http://<quay-server.example.com>/api/v1/user/autoprunepolicy/8c03f995-ca6f-4928-b98d-d75ed8c14859
+$ curl -X DELETE -H "Authorization: Bearer <access_token>" http://<quay-server.example.com>/api/v1/<user>/autoprunepolicy/8c03f995-ca6f-4928-b98d-d75ed8c14859
 ----
 +
 .Example output
 +
 [source,terminal]
 ----
 {"uuid": "8c03f995-ca6f-4928-b98d-d75ed8c14859"}
+----
+
+[id="creating-policy-repository-v2-ui"]
+== Creating an auto-prune policy for a repository using the {productname} v2 UI
+
+Use the following procedure to create an auto-prune policy for a repository using the {productname} v2 UI.
+
+.Prerequisites 
+
+* You have enabled the `FEATURE_AUTO_PRUNE` feature.
+
+.Procedure 
+
+. Tag four sample images, for example, `busybox`, that will be pushed to the repository with auto-pruning enabled. For example:
++
+[source,terminal]
+----
+$ podman tag docker.io/library/busybox <quay-server.example.com>/<organization_name>/<repository_name>:test
+----
++
+[source,terminal]
+----
+$ podman tag docker.io/library/busybox <quay-server.example.com>/<organization_name>/<repository_name>:test2
+----
++
+[source,terminal]
+----
+$ podman tag docker.io/library/busybox <quay-server.example.com>/<organization_name>/<repository_name>:test3
+----
++
+[source,terminal]
+----
+$ podman tag docker.io/library/busybox <quay-server.example.com>/<organization_name>/<repository_name>:test4
+----
+
+. Push the four sample images, for example, `busybox`, to the repository with auto-pruning enabled by entering the following commands:
++
+[source,terminal]
+----
+$ podman push <quay-server.example.com>/<organization_name>/<repository_name>:test
+----
++
+[source,terminal]
+----
+$ podman push <quay-server.example.com>/<organization_name>/<repository_name>:test2
+----
++
+[source,terminal]
+----
+$ podman push <quay-server.example.com>/<organization_name>/<repository_name>:test3
+----
++
+[source,terminal]
+----
+$ podman push <quay-server.example.com>/<organization_name>/<repository_name>:test4
+----
+
+. Check that there are four tags in your repository. 
+
+. On the {productname} v2 UI, click *Repository* in the navigation pane.
+
+. Select the name of an organization that you will apply the auto-pruning feature to, for example, `quayadmin/test_organization`. 
```

Comment:
if this is talking about creating repository auto-prune policy, the search in Repository page should be for repository: `namespace/repo_name`